src/components/fx/layout_attributes.js

```diff
@@ -61,6 +61,16 @@ module.exports = {
         editType: 'modebar',
         description: [
             'Determines the mode of hover interactions.',
+            'If `closest`, a single hoverlabel will appear',
```

Comment:
Please put flags between two * and attributes between ` chars.
E.g. this line should be 
```js
'If *closest*, a single `hoverlabel` will appear',
```